app/views/smart_proxies/_openscap_spool.html.erb

```diff
@@ -0,0 +1,7 @@
+<span>
+  <% if last_error %>
+    <%= _('%s ago') % time_ago_in_words(Time.at(last_error["timestamp"])) %>
```

Comment:
the time information does not say much, could you please add a link to logs? this seems to work just fine
```
<%= link_to _('See logs for more information', '#logs') %>
```

also with following js you could filter relevant logs

```js
var string = 'Failed to parse Arf Report';
$("div#table-proxy-status-logs_filter input").val(string);
var table = $('#table-proxy-status-logs').DataTable();
table.search(string).draw();
```

It would be also good to add a help icon with tooltip explaining what is "spool error". Bonus points for sending a PR to core that would add a JS function so plugins could generate such links easier.